resources/views/admin/jexactyl/approvals.blade.php

```diff
@@ -41,6 +35,19 @@
                                     <p class="text-muted"><small>Determines whether users must be approved by an admin to use the Panel.</small></p>
                                 </div>
                             </div>
+                            <div class="form-group col-md-4">
+                                <label class="control-label" for="webhook">Webhook</label>
+                                <select name="webhook" id="webhook" class="form-control">
+                                    <option @if ($webhook == 'false') selected @endif value="false">Disabled</option>
+                                    <option @if ($webhook == 'true') selected @endif value="true">Enabled</option>
+                                </select>
+                                <p class="text-muted"><small>Enable this to send a Discord Webhook when a user registers.</small></p>
+                            </div>
+                            <div class="form-group col-md-4 @if ($webhook === 'true') visible @else hidden @endif">
+                                <label class="control-label" for="webhook_url">Webhook URL</label>
+                                <input name="webhook_url" id="webhook_url" class="form-control" value="{{ $webhook_url }}">
+                                <p class="text-muted"><small>Enable this to send a Discord Webhook when a user registers.</small></p>
```

Comment:
Replace this with 'Provide the Webhook URL to use when a user needs to be approved.'